src/components/Client.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { makeStyles } from 'tss-react/mui';
+import classNames from 'classnames';
+import { Avatar, Box, CircularProgress, Divider, Typography } from '@mui/material';
+import { getClient, removeClient } from '../store/clients/actions';
+import { selectClient } from '../store/clients/selectors';
+import { theme, getFirstLetters } from '../utils';
+import PostService from '../api/PostServise';
+import { Description } from './Description';
+import { useFetching } from '../hooks';
+import { Section } from './Section';
+import { BackButton } from './index';
+
+const useStyles = makeStyles()(() => ({
+  flex: {
+    display: 'flex',
+  },
+  root: {
+    overflow: 'hidden',
+    height: '100%',
+    alignItems: 'center',
+    width: '100%',
+    justifyContent: 'center',
+  },
+  avatar: {
+    width: '160px',
+    height: '160px',
+    marginRight: theme.spacing(10),
+    background: 'pink',
+    [theme.breakpoints.down('md')]: {
+      marginBottom: theme.spacing(2),
+    },
+  },
+  details: {
+    [theme.breakpoints.down('md')]: {
+      display: 'block',
+    },
+  },
+}));
+
+interface Props {
+  clientID: number | null;
+  onRemoveClientID: () => void;
+}
+
+export const Client: FC<Props> = ({ clientID, onRemoveClientID }) => {
+  const dispatch = useDispatch();
+  const { classes } = useStyles();
+  const client = useSelector(selectClient);
+
+  const [fetchClientsById, isClientIdLoading, clientIdError] = useFetching(async (id: number) => {
+    const response = await PostService.getClient(id);
+    dispatch(getClient(response.data));
+  });
+
+  useEffect(() => {
+    if (clientID === null) {
+      dispatch(removeClient(null));
+      return;
+    }
+    fetchClientsById(clientID);
+  }, [clientID]);
+
+  if (!client) return null;
+  return (
+    <>
+      <BackButton onRemoveClientID={onRemoveClientID} />
+      <div className={classNames(classes.root, classes.flex)}>
+        <div className={classNames(classes.details, classes.flex)}>
+          <Avatar alt="Remy Sharp" src={client.general.avatar} className={classes.avatar}>
```

Comment:
Please change to correct the naming for `alt`

Reply:
Done